Cargo.toml

```diff
@@ -188,3 +189,5 @@ authority-selection-inherents = { path = "primitives/authority-selection-inheren
 session-manager = { path = "primitives/session-manager", default-features = false }
 sp-sidechain = { path = "primitives/sidechain", default-features = false }
 chain-params = { path = "primitives/chain-params", default-features = false }
+pallet-native-token-management = { path = "pallets/native-token-management", default-features = false }
+sp-native-token-management = { path = "primitives/native-token-management", default-features = false }
```

Comment:
"native token" is very broad term but this pallet and associate primitives are very specific. I don't have a better candidate for the name currently, but maybe you have.

Reply:
Maybe "{sp,pallet}-cross-chain-liquidity"? I don't have a solid one either.